### ISSUE TITLE
AP: Rename IO::Handle.slurp-rest to just .slurp

### ISSUE THREAD

**Issue by zoffixznet** [zoffixznet/IOwesomeness#9]

**Comment by molecules** [comment]
I totally agree. `.slurp-rest` does not click with my brain, but using `.slurp` is just naturally what someone would expect.

**Comment by AlexDaniel** [comment]
I am quite interested in knowing why it was called like this in the first place. For example:
https://irclog.perlgeek.de/perl6/2014-11-07#i_9622383
and
https://irclog.perlgeek.de/perl6/2014-11-06#i_9621920
